node/package.json

```diff
@@ -28,7 +28,7 @@
     "@types/atob": "^2.1.2",
     "@types/jest": "^27.0.3",
     "@types/node": "^12.12.21",
-    "@vtex/api": "6.45.19",
+    "@vtex/api": "6.45.21",
```

Comment:
This was done automatically when I linked the app, so it's probably best to update it.